tests/test-stbt-py.sh

```diff
@@ -613,29 +595,6 @@ test_global_use_old_threading_behaviour() {
 }
```

Comment:
I think it's probably worth keeping the parts of these tests that I've kept.  Perhaps the tests need renaming. @drothlis: What do you think?